src/components/Button/index.tsx

```diff
@@ -365,3 +365,33 @@ export function ButtonRadioChecked({ active = false, children, ...rest }: { acti
     )
   }
 }
+
+export const MediumButton = styled.button`
+  align-items: center;
+  background-color: ${({ theme }) => theme.backgroundInteractive};
+  border-radius: 16px;
+  border: 0;
+  color: ${({ theme }) => theme.textPrimary};
+  cursor: pointer;
+  display: flex;
+  flex-direction: row;
+  font-weight: 600;
+  gap: 12px;
+  justify-content: center;
+  padding: 16px;
+  transition: 150ms ease background-color opacity;
+
+  :disabled {
```

Comment:
Does disabled have `cursor: default`?

Reply:
good catch. This should get cursor: pointer by default since it's a button, but for some reason I'm not getting the user agent styles I'm expecting when I use it, so I've set it manually. I'd need to add default for disabled here to override that.